misk/src/test/kotlin/misk/web/WebSocketsTest.kt

```diff
@@ -73,4 +90,20 @@ class EchoWebSocket @Inject constructor() : WebAction {
       override fun toString() = "EchoListener"
     }
   }
+
+  @ConnectWebSocket("/echo-logging-disabled-by-env")
+  @LogRequestResponse(
+    bodySampling = 1.0,
+    errorBodySampling = 1.0,
+    disabledEnvironments = ["testing"]
```

Comment:
I'd rather:
a) use `excludeDeployments`, but that's really a nit.
b) use an `Array<Deployment>` so it's typed.

Reply:
> use an Array<Deployment> so it's typed.

Annotations can only have primitive type params. (Also we'd have to add a dependency on wisp-deployments, though maybe that would be fine)

> use excludeDeployments, but that's really a nit.

Do you still feel that way even if the type remains a string? I would find it confusing, personally.

Reply:
> As for deployments vs environments: do you still feel that way even if the type remains a string? I would find it confusing, personally.

I'd keep environments given we can't use the type.